integration/js/README.md

```diff
@@ -142,12 +142,31 @@ Run the test:
 ```shell
 $KITE_HOME/scripts/mac/path/r configs/<test_name>.config.json
 ```
-Steps to add test file to Sauce Labs virtual machine
-To create a fake video or audio stream you need the media file on Sauce Labs virtual machine before running your test. For that you can use a pre-run executable script to download file from a public storage to the virtual machine running your tests. 
-- Create a pre-run script by following the steps given on this link: https://docs.saucelabs.com/web-apps/automated-testing/selenium/pre-run-executables/#downloading-files-to-a-vm-prior-to-testing. 
-    On sauce labs virtual machines files are stored in the below folder locations
-    Windows = C:\Users\Administrator\Downloads
-    Mac = /Users/chef/Downloads
-    Linux = /home/chef/Downloads
-- Upload the pre-run executable to Sauce Labs storage by using Sauce Labs API methods: https://docs.saucelabs.com/dev/api/storage/
-- Add the pre-run to Sauce configs in your code like this prerun: 'storage:filename=filename.ext'. You can also use storage id of the file location. 
+## Testing against audio/video stream in Saucelabs
+Some tests might require testing against a custom audio/video stream. To test against a custom audio or video stream, you need the media file on Sauce Labs virtual machine before running your test. For that you can use a pre-run executable script to download file from a public storage to the virtual machine before running your tests. For our use case, a pre-run executable script is a minimal `bash` script that will use `curl` to install the resources on the Sauce Labs VM. Pre-run executable scripts can be used for several different use cases like changing VM configurations and browser settings. In our use case, we are using the pre-run executable to download audio/video assets. More information on pre-run scripts can be found [here](https://docs.saucelabs.com/web-apps/automated-testing/selenium/pre-run-executables/).
```

Comment:
How about simplifying like the following?

> To test against a custom audio/video stream, you need a media file on Sauce Labs' virtual machine before running your test. To do so, you can use a pre-run executable script to download a media file from public storage to the virtual machine. In our use case, we use a `bash` script to download audio/video assets on the virtual machine. You can use pre-run executable scripts for other use cases, such as changing VM configurations and browser settings. See [Using Pre-Run Executables](https://docs.saucelabs.com/web-apps/automated-testing/selenium/pre-run-executables/) for more information.

Reply:
Simplified the first paragraph